src/components/CollapsableItem.tsx

```diff
@@ -36,7 +36,7 @@ const StyledCollapsableHeader = styled.div`
     padding-bottom: 1px;
 `;
 
-const StyledCollapsableItem = styled.div`
+export const StyledCollapsableItem = styled.div`
```

Comment:
Sorry, but it makes code dirty. In time when you want to export part of component with styles you must create new component. Component style is private API.

Reply:
fixed